chatgpt_wrapper/chatgpt.py

```diff
@@ -1,26 +1,29 @@
+""" AsyncChatGPT class
+"""
 import atexit
 import base64
 import json
 from json.decoder import JSONDecodeError
-import operator
 import time
 import uuid
-import os
 import logging
 import shutil
-from functools import reduce
-from time import sleep
+import time
+import datetime
+import threading
 from typing import Optional
 from playwright.sync_api import sync_playwright
 from playwright._impl._api_structures import ProxySettings
+# from . import error
+import error
 
 RENDER_MODELS = {
     "default": "text-davinci-002-render-sha",
     "legacy-paid": "text-davinci-002-render-paid",
     "legacy-free": "text-davinci-002-render"
 }
 
-DEFAULT_CONSOLE_LOG_LEVEL = logging.ERROR
+DEFAULT_CONSOLE_LOG_LEVEL = logging.DEBUG
```

Comment:
Can't do this, the console level needs to remain at `ERROR`, the user can pass the `--debug-log` argument to get more detailed logs written to a file.

Reply:
Reverted.